src/pom.xml

```diff
@@ -143,11 +143,10 @@
         <slf4j.version>1.7.25</slf4j.version>
         <jackson.version>2.9.3</jackson.version>
         <opencsv.version>4.1</opencsv.version>
-        <ehcache.version>3.4.0</ehcache.version>
+        <ehcache.version>3.5.0</ehcache.version>
```

Comment:
You use `JCacheRegionFactory` in db context for `"hibernate.cache.region.factory_class`. Is ecache needed anymore?

Reply:
Yes, ehcache brings a JCache implementation.